src/grt/vhpi_user.h

```diff
@@ -620,7 +620,7 @@ typedef enum {
 
 #ifdef VHPIEXTEND_INT_PROPERTIES
         VHPIEXTEND_INT_PROPERTIES
-        
+
```

Comment:
Why is there a new empty line?

Reply:
I don't think this is a new empty line, just trailing blank space removed.

Reply:
Oh, now I get it.